### PR TITLE
Feature task estimation

### PR DESCRIPTION
**Pull request by 4kh**
Here is the task estimation part of the documenation
Don't hesite to comment or propose some additional content

### CODE REVIEW

**Review by valotg — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,122 @@
...
+
+The subject must be clarified to avoid confusion and misunderstanding during the project development. To ensure the project is understood by the whole team, meetings must be organized frequently during which:
+
+- [ ] Requirements should be defined or redefined
+- [ ] Long-term, mid-term and short-term objectives are then defined or redefined
+- [ ] In some case, a new specific feature asked by the client may be defined
+- [ ] All the tasks that need to be done are defined
+- [ ] Team members that will be involved
+- [ ] Project deadline might be defined
+
```
Comment (Arakhaa) [inline README.md:21]: I like this to do list

File: README.md [hunk 2]
```diff
@@ -2,6 +2,122 @@
...
+### SCRUM method
+
+**What's SCRUM method ?**
+
+Scrum is an agile method dedicated to "project management". This management method, or rather this Project Management Framework, aims to improve the productivity of your team.
+This method includes several roles that the whole team must play and it's very helpful to the task estimation.
+
+### SCRUM method roles
+
```
Comment (Arakhaa) [inline README.md:51]: Maybe add an image to understand the "hierachy" in SCRUM methodology ?

File: README.md [hunk 3]
```diff
@@ -2,6 +2,122 @@
...
+Within a Scrum team, the Product Owner makes the link between the business part and the technical part of the project, here are his main objectives: 
+
+- Business expert, defines functional specifications
+- Prioritizes the features to be developed or corrected
+- Validates the developed functionalities
+- Plays the role of the customer
+
+### Sprints
+
+The Scrum life cycle is punctuated by iterations of a few weeks, the sprints.
```
Comment (Arakhaa) [inline README.md:82]: Thanks to our Business Modeling lecture, you could specify the time of sprint ? (between 5 days to 2 weeks in general)

**Review by Arakhaa — CHANGES_REQUESTED** [review]
Take a look at my comments before merging

**Review by EtiennePASSOT — APPROVED** [review]

**Comment by Arakhaa** [comment]
Thanks for the changes

**Pull request merged**